pkg/cmd/step/create/step_create_task.go

```diff
@@ -231,9 +231,16 @@ func (o *StepCreateTaskOptions) Run() error {
 		}
 	}
 
+	if o.VersionResolver == nil {
```

Comment:
@abayer we need to make sure the version resolver gets initialised also for the meta pipeline (when the effective pipeline config exists)